index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head> 
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;400&display=swap" rel="stylesheet">
+    <title>Muneeb's protfolio</title>
+</head>
+<body>
+    <!--This is the header section-->
+    <header>
+        <nav>
+            <div class="Navbar">
+            <a href="#"><img src="req/mylogo.png" alt="logo" id="mylogo"></a>
+            <!--Nav bar button-->
+            <img id="logo" src="req/Union.png" alt="logo">
+            </div>
+        </nav>
+    </header>
+    <!--This is Headline-->
+    <section class="headline">
+    
+        <h1 id="Headline-Title">I’m Muneeb<br>
+            Glad to <br>see you!</h1>
+        <p id="Headline-Text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.  </p>
+    <div class="socialmedia">
+        <h3 class="helpingtext">LET'S CONNECT</h3>
+        <!--social media icons-->
+        <div class="belowtext">
+        <ul>
+            <li>
+                <a href="#"><img src="req/twitter.png" alt="twitter"></a>
+            </li>
+            <li>
+                <a href="#"><img src="req/linkedin.png" alt="linkedin"></a>
```

Comment:
Your project looks great. However, the image for the linkedin logo does not show up yet. This is because the file is named `Linkedin.png` while the URL has `linkedin.png`. Kindly make this modification and your project will be ready for approval.

![image](https://user-images.githubusercontent.com/20554583/219602180-a30e7cc3-0c38-4ed3-b20d-85a298639e50.png)

Reply:
@emmiiorji I used the correct link i don't know what happened on your site. in my live server this link will not broke

Reply:
![image](https://user-images.githubusercontent.com/118625824/219621163-8167c0b9-ac7a-47ff-8461-c1a749d8c14a.png)